test/iron-overlay-behavior.html

```diff
@@ -100,11 +100,22 @@
           assert.equal(getComputedStyle(overlay).display, 'none', 'overlay starts hidden');
         });
 
-        test('overlay open by default', function() {
+        test('overlay open by default', function(done) {
           overlay = fixture('opened');
-          runAfterOpen(overlay, function() {
+          overlay.addEventListener('iron-overlay-opened', function() {
```

Comment:
changed to this, because `runAfterOpen` is actually calling `overlay.open()` which defeats the purpose of this test